CONTRIBUTING.rst

```diff
@@ -43,6 +43,15 @@ To run the tests and view coverage, execute:
 
    pytest --cov=aatoolbox
```

Comment:
I would just swap out this command with the one you specified below, you can then give a warning that "If you run it directly with pytest, you may get the following error..." or something

Reply:
Good idea! Addressed!